src/main/java/org/zalando/nakadi/domain/Storage.java

```diff
@@ -70,6 +75,7 @@ public String toString() {
     }
 
     private String id;
+    @JsonProperty("storage_type")
```

Comment:
I don't really like the way we do that (but of course it's up to team)
I thought that approach with different View and Model objects is much better. For example, from current code I can not figure out why configuration objects should be mapped to `kafka_configuration` value.